dotCMS/src/main/java/com/dotcms/graphql/business/PageAPIGraphQLTypesProvider.java

```diff
@@ -108,6 +110,8 @@
 
         pageFields.put("layout", new TypeFetcher(GraphQLTypeReference.typeRef("Layout"),
                 new LayoutDataFetcher()));
+        pageFields.put("containers", new TypeFetcher(list(GraphQLTypeReference.typeRef("Container")),
```

Comment:
Codacy found an issue: [The String literal "containers" appears 5 times in this file; the first occurrence is on line 113](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6090726)